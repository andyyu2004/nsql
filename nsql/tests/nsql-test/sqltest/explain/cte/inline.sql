EXPLAIN WITH t AS NOT MATERIALIZED (
    SELECT
        1
)
SELECT
    *
FROM
    t;
