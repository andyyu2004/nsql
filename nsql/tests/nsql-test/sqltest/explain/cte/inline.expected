 projection (<compiled expr>) 
   dummy scan                 
                              